client/commands/seeds/export.go

```diff
@@ -40,5 +44,20 @@ func exportSeed(cmd *cobra.Command, args []string) error {
 	}
 
 	// now get the output file and write it
-	return seed.Write(path)
+	return writeSeed(seed, path)
+}
+
+func writeSeed(seed certifiers.Seed, path string) (err error) {
+	var f *os.File
+	f, err = os.Create(path)
+	if err == nil {
+		stream := json.NewEncoder(f)
+		err = stream.Encode(seed)
+		f.Close()
```

Comment:
Yeah, good call on the os.IsExist... i used it for dir detection before and just figured it worked the same.

And if I do this re-work, I can remove the IsExist check and then the `err != nil` early return is a define best choice.

Thanks for the feedback.